chart/values.yaml

```diff
@@ -263,8 +263,6 @@ migrationWaiter:
       cpu:
       # memory defines the memory limit for the registry server
       memory: 50Mi
-      # ephemeralStorage defines the ephemeral storage limit for the verifier server
-      ephemeralStorage: 100Mi
```

Comment:
Where is this value used?

Reply:
it's not :D

Reply:
I understand why you remove it then, but that does not fix the error?